plugins/flyctl/fly.go

```diff
@@ -0,0 +1,22 @@
+package flyctl
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func FlyctlCLI() schema.Executable {
+	return schema.Executable{
+		Name:      "Flyctl",
+		Runs:      []string{"fly"},
```

Comment:
Does `Executable.Runs` not support more than one command at this time? 

I tried `Runs:      []string{"fly", "flyctl"},`  but after building the local binary, setting up the plugin fails:

```
➜  shell-plugins git:(add/flyctl) ✗ make flyctl/build
# BETA NOTICE: The plugin ecosystem is in beta and is subject to change.
# You may have to update or recompile your local builds every now and then to keep them
# compatible with the 1Password CLI updates.

go build -o ~/.op/plugins/local/flyctl -ldflags="-X 'main.PluginName=flyctl'" ./cmd/contrib/build/
➜  shell-plugins git:(add/flyctl) ✗ op plugin list
SOURCE                USAGE         NAME                  REQUIRED FIELDS
Local Build           fly flyctl    Fly.io                Token
1Password Registry    aws           AWS                   Access Key ID, Secret Access Key
.
.
.
.
.
.
➜  shell-plugins git:(add/flyctl) ✗ op plugin init fly
[ERROR] 2023/01/12 02:26:54 unknown plugin: fly
➜  shell-plugins git:(add/flyctl) ✗ op plugin init flyctl
[ERROR] 2023/01/12 02:26:56 unknown plugin: flyctl
```

@SimonBarendse, it's at this point, I scheduled a Calendly pairing session but managed to resolve this by completely removing `flyctl`.

Reply:
This should be addressed by https://github.com/1Password/shell-plugins/pull/141/commits/01c6f85fc3113c02bbcefdde6e01d1d53e6576e8.